python-sdk/tests_integration/astro_deploy/Dockerfile.single_worker

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/astronomer/astro-runtime:8.1.0-base
+FROM quay.io/astronomer/astro-runtime:10.0.0-base
```

Comment:
Did we manually update this? 
I see the version was updated for the other Dockerfile in PR https://github.com/astronomer/astro-sdk/pull/2081/, but not for this file. Are we missing something?

Reply:
the PR https://github.com/astronomer/astro-sdk/pull/2081 was created by this job but it ran only on schedule intervals I updated it to run on push to test the changes. This line in the current PR is a manual change so that all docker file will point to same version once the PR will merge and going forward the job will create PR

Reply:
okay, should we remove this manual change and re-run the job once to verify it works fine? Will leave it to you to decide.